src/antsibull_docs/data/docsite/plugin.rst.j2

```diff
@@ -291,14 +293,19 @@ Attributes
 
 .. Notes
 
-{% if doc['notes'] -%}
+{% if doc['notes'] or (has_positional_parameters and has_keyword_parameters) -%}
 Notes
 -----
 
 .. note::
-{%   for note in doc['notes'] %}
+{%   if has_positional_parameters and has_keyword_parameters %}
+   - When keyword and positional parameters are used together, positional parameters must be listed before keyword parameters: ``input | @{plugin_name}@(positional1, positional2, key1=value1, key2=value2)``.
```

Comment:
The syntax you use here is only for filters, but not for tests or lookups. Also most importantly, it is **not for modules or any other kind of plugins**. Right now you are adding this note for every module or plugin that has options.
```suggestion
   - When keyword and positional parameters are used together, positional parameters must be listed before keyword parameters:
{%     if plugin_type == 'filter' %}
     ``input | @{plugin_name}@(positional1, positional2, key1=value1, key2=value2)``.
{%     elif ... %}
        ...
{%     endif %}
```